airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_core.py

```diff
@@ -1151,11 +1152,21 @@ async def test_airbyte_trace_message_on_failure(self, connector_config, inputs:
                 ConfiguredAirbyteStream.construct(
                     stream=AirbyteStream(
                         name="__AIRBYTE__stream_that_does_not_exist",
+                        namespace="__AIRBYTE__namespace_that_does_not_exist",
                         json_schema={"type": "object", "properties": {"f1": {"type": "string"}}},
                         supported_sync_modes=[SyncMode.full_refresh],
+                        source_defined_primary_key=[],
+                        source_defined_cursor=False,
+                        default_cursor_field=[],
+                        is_resumable=False,
                     ),
-                    sync_mode="INVALID",
-                    destination_sync_mode="INVALID",
+                    sync_mode=SyncMode.full_refresh,
+                    destination_sync_mode=DestinationSyncMode.append,
+                    primary_key=[],
+                    cursor_field=[],
+                    minimum_generation_id=1,
+                    generation_id=1,
+                    sync_id=1,
```

Comment:
Without this fix, source connectors built with the bulk CDK will fail unexpectedly. The Bulk CDK performs schema validation on the connector CLI inputs, so if these fields are missing, its execution fails very early.

Reply:
Would you mind adding a comment with a link to the bulk CDK validation?